pkg/kube/nsmounter

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#
+# nsmounter: script called by the longhorn-csi-plugin used to find the correct
+#   PID to nsenter into and mount RWX volumes for longhorn
+#
+
+PROC_DIR="/proc"
+
+os_distro_eve="eve"
+os_distro=""
+
+get_os_distro() {
+  local -r version_info=$(< $PROC_DIR/version)
+
+  [[ $version_info =~ $os_distro_eve ]] && os_distro=$os_distro_eve
+}
+
+target_pid=1
+
+get_pid() {
```

Comment:
Is this function really needed? `pidof` cannot be used to get the PID of the cluster-init.sh script?

Reply:
This is a slightly trimmed version of one containing a patch to also find the correct pid on eve.  It will be upstreamed soon

Reply:
The longhorn pod doesn't have pidof but has pgrep.  I'll commit a shortened nsmounter to resolve this